src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;0,700;1,400&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__img-box">
+        <img
+          src="./images/imac.jpeg"
+          alt="product-img"
+          class="product-img"
+        >
+      </div>
+      <div class="card__product-name">Моноблок APPLE A1419 iMac
+        <br>27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card__product-code">Код товара: 195434</div>
+      <div class="star-block card__stars">
+        <div class="stars star-block__stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div>Отзывов: 5</div>
+      </div>
+      <div class="product-price card__price">
+        <div class="product-price__price">Цена:</div>
+        <div class="product-price__value">69 999 грн</div>
+      </div>
+      <a
+        href="/"
+        class="link"
+        data-qa="hover"
+      >
+        <div class="button">Купить</div>
```

Comment:
```suggestion
        Купить
```
Тут можна обійтись і без цього діва